src/Events/WithDelay.php

```
@@ -24,16 +24,17 @@
 
 namespace Bottledcode\DurablePhp\Events;
 
+use DateTimeImmutable;
 use Ramsey\Uuid\Uuid;
 
 class WithDelay extends Event implements HasInnerEventInterface
 {
-    public function __construct(string $eventId, public \DateTimeImmutable $fireAt, public Event $innerEvent)
+    public function __construct(string $eventId, public DateTimeImmutable $fireAt, public Event $innerEvent)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$eventId'. (Unused Code Rules)

(UnusedFormalParameter)